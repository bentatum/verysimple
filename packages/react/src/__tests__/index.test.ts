import * as index from "..";

describe("top level exports", () => {
  it("matches snapshot", () => {
    expect(Object.keys(index)).toMatchSnapshot();
  });
});
