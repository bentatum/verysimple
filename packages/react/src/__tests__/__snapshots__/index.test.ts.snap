// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`top level exports matches snapshot 1`] = `
[
  "ButtonBase",
  "Card",
  "Dialog",
  "DialogTitle",
  "DialogContent",
  "DialogActions",
  "DialogCloseButton",
  "FilledButtonBase",
  "FilledIconButton",
  "GradientBorder",
  "buttonShadowClassNames",
  "buttonTextClassNames",
  "fieldBorderRadiusClassNames",
  "fieldPaddingClassNames",
  "fieldSizeClassNames",
  "gradientBorderStyles",
  "iconButtonWidthClassNames",
  "shadowTransitionClassNames",
  "statusColorClassNames",
  "IconButton",
  "NormalIconButton",
  "OutlinedButtonBase",
  "OutlinedIconButton",
  "Spinner",
]
`;
